packages/parser/src/repetition.ts

```
@@ -0,0 +1,244 @@
+      default:
+        return expression;
+    }
+  } else {
+    // Quantified repetition
+    return {
+      type: 'Quantified' as const,
+      expression,
+      min: operator.min,
+      max: operator.max
+    } as Quantified;
+  }
```

Biome (1.9.4)

[error] This else clause can be omitted because previous branches break early.

Unsafe fix: Omit the else clause.


(lint/style/noUselessElse)